web/tests/hotkey.test.cjs

```diff
@@ -204,8 +197,6 @@ run_test("mappings", () => {
     assert.equal(map_down("p", false, false, false, true).name, "toggle_compose_preview"); // Alt + P
 
     // More negative tests.
-    assert.equal(map_down("/"), undefined);
-    assert.equal(map_press("Escape"), undefined);
```

Comment:
These two were earlier passing because `"/"` was handled by `keypress` and `"Escape"` by `keydown`. Now, both are handled by `keydown` so just removed it from here as we already test them earlier in this test file.